mastering-plone/events.md

```diff
@@ -2,23 +2,23 @@
 
 # Turning Talks into Events
 
-````{sidebar}
-Volto chapter
+````{sidebar} Volto chapter
 ```{figure} _static/volto.svg
 :alt: Volto Logo
 ```
 
 This chapter is about the React frontend Volto.
 
 Solve the same tasks in Plone Classic in chapter {doc}`events_classic`
-````
 
-````{sidebar}
-Get the code! ({doc}`More info <code>`)
+---
+
+**Get the code! ({doc}`More info <code>`)**
+
 Code for the beginning of this chapter:
 
 ```
-# frontent
+# frontend
```

Comment:
`#` is now a heading in `.md` when not in a `code-block`. I think we need to put this into a codefence with triple backticks and set the language to `shell`.

Reply:
And even more important: this should not go into the same code block, as these are commands to be executed in different folders.

Reply:
This is a code block already. I set explicitly the language to `shell`